data-serving/data-service/src/index.ts

```diff
@@ -36,13 +36,11 @@ app.use('/api', apiRouter);
                and collection ${env.DB_COLLECTION}`,
```

Comment:
DB_COLLECTION doesn't seem used anymore?

Reply:
mongoose derives collection names from the first argument in the schema definition, so this is not needed anymore

Reply:
how does that work with having different collections with the same fields, e.g. one for covid19 and one for ebola?

Reply:
Yeah -- I held off on removing the now-obviated params until we settled on an approach. I'll remove this.